content/en/contribute/medic/product-development-process/ux-research-repo/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "UX Research Repository"
+linkTitle: "UX Research Repository"
+weight: 4
+description: >
+  How to keep track of product research
+---
+
+At Medic, we are implementing [baserow.io](http://baserow.io) as a place to keep all our product research. Baserow is very similar to [airtable.com](http://airtable.com), [getgrist.com](http://getgrist.com), and Google Sheets.
+
+Our UX Research Repository is based off of concepts from WeWork’s [Polaris](https://tsharon.medium.com/democratizing-ux-670b95fbc07f) project. You can see an example of the Polaris (which uses Airtable) [here](https://www.airtable.com/universe/expShuhNMi0Oc0xpb/polaris-ux-nuggets).
+
+The Baserow interface can be seen below. The items in the list on the left are “tables” that we have created. On the right is the “Grid”. It looks just like a spreadsheet and you can have multiple views of the same grid.
+
+![The Baserow interface 👆🏼](images/baserow.png)
+
+To gain access to our UX Research Repo, please reach out in the #product-owner-chatter Slack channel.
```

Comment:
How should external people gain (view) access to the UX Research Repo?

```suggestion
To gain access to the UX Research Repo, please reach out in the #product-owner-chatter Slack channel.
```

Reply:
Good question. @craig-landry suggested keeping the Baserow internal to the Medic team. I added a clarification about this in the paragraph above.

Reply:
As a long(er) term goal for the UX Repo, I would love to see a point where we can make it public. Currently it has sensitive information like interview recordings and names and contact information for people we've spoken with. Baserow has functionality that can help us remove or obfuscate data though, so with some effort and careful usage we can probably have a public-facing view available.